swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,203 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+int db_port							= 6379;
+const char* const hostname 			= "localhost";
+const char* const op_name 			= "OP";
+const char* const op_set 			= "SET";
+const char* const op_del 			= "DEL";
+const char* const name_delimiter 	= ":";
+const int el_count = 2;
+
+#define _in_
+#define _out_
+#define _inout_
+
+typedef struct _sonic_db_item_t {
+	string op_val;
+	string hash_name;
+	std::vector<FieldValueTuple> fvVector;
+}sonic_db_item_t;
+
+void usage(char **argv)
+{
+    cout <<"Usage: " << argv[0] << " json_file_path\n";
+}
+
+void dump_db_item_cout(_in_ sonic_db_item_t& db_item)
+{
+	cout << "db_item [\n";
+	cout << "operation: " << db_item.op_val << "\n";
+	cout << "hash: " << db_item.hash_name << "\n";
+	cout << "[\n";
+	for(auto& fv: db_item.fvVector) {
+		cout << "field: " << fvField(fv);
+		cout << "value: " << fvValue(fv) << "\n";
+	}
+	cout << "]\n";	
+	cout << "]\n";	
+}
+void dump_db_item(_in_ sonic_db_item_t& db_item)
+{
+	SWSS_LOG_NOTICE("db_item: [\n");
+	SWSS_LOG_NOTICE("operation: %s", db_item.op_val.c_str());
+	SWSS_LOG_NOTICE("hash: %s\n", db_item.hash_name.c_str());
+	SWSS_LOG_NOTICE("fields: [\n");
+	for(auto& fv: db_item.fvVector) {
+		SWSS_LOG_NOTICE("field: %s", fvField(fv).c_str());
+		SWSS_LOG_NOTICE("value: %s\n", fvValue(fv).c_str());
+	}
+	SWSS_LOG_NOTICE("]\n");	
+	SWSS_LOG_NOTICE("]\n");
+}
+
+bool write_db_data(_in_ std::vector<sonic_db_item_t>& db_items)
+{
+    DBConnector db(APPL_DB, hostname, db_port, 0);
+#ifdef _DUMPT_TO_COUT_
+	for (sonic_db_item_t& db_item : db_items) {
+		dump_db_item_cout(db_item);
+	}
+#endif //_DUMPT_TO_COUT_	
```

Comment:
I suggest to remove this define and this value. You can have private debug in your private copy.